infrastructure_files/getting-started.sh

```
@@ -345,11 +345,34 @@ initialize_default_values() {
+  DATASTORE_ENCRYPTION_KEY=""
 
+  if openssl &> /dev/null
+  then
+    NETBIRD_RELAY_AUTH_SECRET=$(openssl rand -base64 32 | sed "$SED_STRIP_PADDING")
+    # Note: DataStoreEncryptionKey must keep base64 padding (=) for Go's base64.StdEncoding
+    DATASTORE_ENCRYPTION_KEY=$(openssl rand -base64 32)
+  else
+    SHOULD_CLEAN_OPENSSL_IMAGE=true
+    OPENSSL_IMAGE=$(docker images --filter=reference="alpine/openssl:*" --format "{{.Repository}}:{{.Tag}}" | head -n 1)
+    
+    if [ -z $OPENSSL_IMAGE ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)